### PR TITLE
fix: Help and comments on `UserPassword.pwrecover`

### PR DESCRIPTION
**Pull request by phorward**
This improves PR #1033 for viur-core 3.6

### CODE REVIEW

**Review by ArneGudermann — DISMISSED** [review]

File: src/viur/core/modules/user.py [hunk 1]
```diff
@@ -281,13 +281,13 @@ class LostPasswordStep2Skel(skeleton.RelSkel):
             params={
                 "tooltip": i18n.translate(
                     key="viur.modules.user.userpassword.lostpasswordstep2.recoverykey",
-                    defaultText="The key is expired. Please try again",
-                    hint="Shown when the user needs more than 15 minutes to paste the key"
-                )
+                    defaultText="Please enter the validation key you've received via e-mail.",
+                ),
```
Comment (sveneberth) [inline src/viur/core/modules/user.py:285]: Why did you remove the hint?

**Review by sveneberth — CHANGES_REQUESTED** [review]

File: src/viur/core/modules/user.py (hunk 1 above)
Comment (phorward) [reply src/viur/core/modules/user.py]: r u happy now?

**Review by sveneberth — APPROVED** [review]
